sympy/parsing/c/c_parser.py

```diff
@@ -478,16 +488,29 @@ def transform_string_literal(self, node):
             pass
 
         def transform_character_literal(self, node):
-            #TODO: No string Type in AST
-            #type =
-            #try:
-            #    value = next(node.get_tokens()).spelling
-            #except (StopIteration, ValueError):
+            """Transformation function for character literal
+
+            Used to get the value of the given character literal.
+
+            Returns
+            =======
+
+            val : value
```

Comment:
I think this should be
```
val : str
     Its description
```

Reply:
Thanks for suggestion! I will change it as requested.